### DISCUSSION TITLE
Tanix W2 4/32

### DISCUSSION THREAD

**Discussion by xgreend** [devmfc/debian-on-amlogic#30] [Q&A]
pls adding support for ddr3 version 
![image_2024-02-17_144859412](https://github.com/devmfc/debian-on-amlogic/assets/95994365/a472399d-23ea-4c4b-9eea-1e399d9df573)

**Comment by devmfc** [comment]
The image does work, it only can’t find rootfs, probably due to an unsupported SD card (to large?). So try another SD card, or better yet, a USB flash drive.

**Comment by flashvnn** [comment]
It work on my device, you should use USB Stick or use SDCard with Usb Adapter

![photo_2024-06-09 13 23 49](https://github.com/devmfc/debian-on-amlogic/assets/1784547/df28ccd2-b521-49d0-a839-347e34aad78f)


![photo_2024-06-09 13 23 54](https://github.com/devmfc/debian-on-amlogic/assets/1784547/a19a6c82-6053-45ec-8a1a-b30b543cca4a)
